distribution/lib/Standard/Base/0.0.0-dev/src/Data/Ordering.enso

```diff
@@ -170,7 +170,7 @@ type Ordering
     Greater
 
     ## Compares to values and returns an Ordering
-    compare : Any -> Any -> Ordering ! (Incomparable_Values | Type_Error)
+    compare : Any -> Any -> Ordering ! Incomparable_Values
```

Comment:
This is a simplification of the API - good for users of `Ordering.compare` that they don't need to think about two different errors.

Reply:
This was possible because https://github.com/enso-org/enso/pull/5998/commits/603ab95a184871a49ac8b21fe5f8bd4122647aca - I decided that if comparing two values with different comparators, `Incomparable_Values` is thrown rather than `Type_Error`. I think this is, indeed, a change for the better.